RiotSwiftUI/Modules/Room/PollHistory/View/PollListItem.swift

```diff
@@ -50,10 +43,14 @@ struct PollListItem: View {
                     .lineLimit(2)
                     .accessibilityLabel("PollListItem.title")
             }
+            .frame(maxWidth: .infinity, alignment: .leading)
             
-            if pollData.winningOption != nil {
+            if pollData.closed {
                 VStack(alignment: .leading, spacing: 12) {
-                    optionView(winningOption: pollData.winningOption!)
+                    let winningOptions = pollData.answerOptions.filter(\.winner)
+                    ForEach(winningOptions) {
+                        optionView(winningOption: $0)
```

Comment:
Personally I'm not a huge fan of using a `func` to make a subview. Would it be worthwhile extracting this into its own `PollListWinningOptionView` or similar?

Reply:
Did better than that!
I reused an existing view we have on the timeline.
I didn't to it since the start since the models were different. Now they aren't. :)
https://github.com/vector-im/element-ios/pull/7293/commits/ff229ff318b81858cc55c166206356af0f7cfcca

Reply:
Nice! 😎